src/Microsoft.AspNetCore.Sockets.Abstractions/IConnection.cs

```diff
@@ -10,9 +10,9 @@ namespace Microsoft.AspNetCore.Sockets.Client
 {
     public interface IConnection
     {
-        Task StartAsync();
-        Task SendAsync(byte[] data, CancellationToken cancellationToken);
-        Task DisposeAsync();
+        Task StartAsync(CancellationToken cancellationToken = default);
```

Comment:
Does the interface have to have cancellationTokens set to default? Because it shouldn't

Reply:
Yeah, that did feel a bit weird to me.